containers/terraform/Dockerfile

```diff
@@ -47,7 +47,11 @@ RUN curl -sL https://bootstrap.pypa.io/get-pip.py | python3 && \
     pip3 install pyyaml==5.1.1 awscli && \
     gem install hiera hiera-eyaml
 
-COPY vars_template.yaml /opt/legion/
+# Install awscli
+RUN pip3 install awscli
```

Comment:
But we're already installing `awscli` in previous `RUN` :)

Reply:
thanks! removed :)